### PR TITLE
Rs 2022 08 04

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Ciel/Day 5/05_01_CIEL05_2_2B.txt [hunk 1]
```diff
@@ -121,22 +129,23 @@ There's no way I could reach him.
...
 -- Page 369, Offset 11848.
 -- その逆境を、彼女は腕の一振りで<覆|くつがえ>した。
-She sweeps away such adversity in a single swing.
+She sweeps away such adversity with a single swing of her arm.
 }
 [sha:5fd29ffb37804f98c93adc5a7635f0abce37dc5c]{
 -- Page 369, Offset 11849.
 -- 細い針のような剣を五連射。
-Five consecutive shots of a blade, thin as a needle.
+She fires off five needle-thin blades in quick succession.
```
Comment (yk1512) [inline script/Ciel/Day 5/05_01_CIEL05_2_2B.txt:142]: Five needle-thin blades are fired off in quick succession.

File: script/Ciel/Day 5/05_01_CIEL05_2_2B.txt [hunk 2]
```diff
@@ -121,22 +129,23 @@ There's no way I could reach him.
...
 }
 [sha:0a7c50bfba76fc0a4d648a32dc1215b2d96851c4]{
 -- Page 369, Offset 11850.
 -- 一息のうちに左右合わせて10体もの死者を串刺しにし、停止させ、そのただ中をすり抜け、立ち尽くすヴローヴへ肉薄する。
-In the blink of an eye, a total of ten zombies on both sides are skewered and brought to a halt. She quickly makes her way through them, closing in on the motionless Vlov.
+In the blink of an eye, at least ten of the Dead milling around Vlov are skewered and brought to a halt.
+ She slips right through that opening, closing in on the motionless figure of Vlov.
```
Comment (yk1512) [inline script/Ciel/Day 5/05_01_CIEL05_2_2B.txt:148]: closing in on his motionless figure.

File: script/Ciel/Day 5/05_01_CIEL05_2_2B.txt [hunk 3]
```diff
@@ -178,42 +187,44 @@ The gas contained in their stomach ignites in response to Vlov's signal.
...
 -- Page 374, Offset 11864.
 -- 自動的に、迫り来る外敵を攻撃しているだけ。
-He simply reflexively attacked the approaching enemy.
+All he did was reflexively attack the approaching enemy.
 }
 [sha:0910393091200b51229705eff18b376f913ded8d]{
 -- Page 375, Offset 11865.
 -- 考えろ。これが最後の行動だ。
-Think. This is my final move.
+Think. I've only got once chance to make a move.
```
Comment (yk1512) [inline script/Ciel/Day 5/05_01_CIEL05_2_2B.txt:227]: one

**Review by yk1512 — APPROVED** [review]

File: script/Ciel/Day 5/05_01_CIEL05_2_2B.txt [hunk 4]
```diff
@@ -1,87 +1,95 @@
...
-%{i}that%{/i} dark hole.
+I can't see the bottom of it.
 }
 [sha:2a69e1eb1337b7d7cea2754ba716d7e663554210]{
 -- Page 361, Offset 11829.
 -- 「二十七祖……！　まさか成り立て……!?　そんな大物がどうして今、よりにもよってこの街に来るんですか……!?」
-"The Twenty-Seven Ancestors...! No way, he's one of them...!? Why is such a big name now, and of all places right here in this city...?!"
+"The Twenty-Seven Ancestors...!
+ You're saying he's one of %{i}them%{/i}...!?
+ Why did a heavyweight like him have to show up in this city now of all times...?!"
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_2_2B.txt:37]: city, now of

File: script/Ciel/Day 5/05_01_CIEL05_2_2B.txt (hunk 1 above)
Comment (Avuxo) [reply script/Ciel/Day 5/05_01_CIEL05_2_2B.txt:142]: current line is fine imo

File: script/Ciel/Day 5/05_01_CIEL05_2_2B.txt [hunk 5]
```diff
@@ -157,18 +166,18 @@ Vlov snaps his fingers.
...
 -- Page 371, Offset 11855.
 -- 先輩が停止させた死者たち。
-The Dead that Senpai brought to a halt.
+The Dead that Senpai brought to a halt―――
 }
 [sha:44477a6f828c12c8ba59490c02aca996199b5ec8]{
 -- Page 371, Offset 11856.
 -- その腹にたまったガスが、ヴローヴの合図に呼応して着火する。
-The gas contained in their stomach ignites in response to Vlov's signal.
+At Vlov's signal, the gas accumulated in their stomachs detonates.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_2_2B.txt:180]: they swallowed small bombs??
image

**Review by Avuxo — APPROVED** [review]

File: script/Ciel/Day 5/05_01_CIEL05_2_2B.txt (hunk 5 above)
Comment (rschlaikjer) [reply script/Ciel/Day 5/05_01_CIEL05_2_2B.txt:180]: sample-43cd97101384eefbbf95d2f25ba064ee

**Pull request merged**